resources/assets/js/helpers.js

```diff
@@ -183,6 +183,11 @@ $.extend(true, laravelValidation, {
             var timeValue = false;
             var fmt = new DateFormatter();
 
+            if (typeof value === 'number' && format == undefined)
```

Comment:
Please can you make the latter check strict (`===`)

Reply:
ok, done